docs/guides/testing.md

```diff
@@ -187,16 +221,64 @@ function Counter() {
 }
 ```
 
-### Testing stores
+```ts
+// components/counter/index.ts
+export * from './counter'
+```
+
+```tsx
+// components/counter/counter.test.tsx
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import { Counter } from './counter'
+
+describe('Counter', () => {
+  test('should render successfully', async () => {
```

Comment:
_should render successfully_ is a poor name for a test,
because _successfully_ doesn't add any more context,
and it omits an important detail of the initial value.

```suggestion
  test('should render with initial count of 1', async () => {
```